swift_static_framework/swift_static_framework.bzl

```diff
@@ -3,42 +3,13 @@
 load("@build_bazel_apple_support//lib:apple_support.bzl", "apple_support")
 load("@build_bazel_rules_swift//swift:swift.bzl", "SwiftInfo", "swift_library")
 
-_DEFAULT_MINIMUM_OS_VERSION = "10.0"
+_DEFAULT_MINIMUM_OS_VERSION = "11.0"
 
 _PLATFORM_TO_SWIFTMODULE = {
-    "ios_armv7": "arm",
     "ios_arm64": "arm64",
```

Comment:
What’s the reasoning for removing armv7 and i386?

Reply:
Because of the removal of iOS 10 support.